nextflow.config

```diff
@@ -101,27 +108,59 @@ try {
 }
 
 profiles {
-  conda { process.conda = "$projectDir/environment.yml" }
+  conda {
+    docker.enabled = false
+    singularity.enabled = false
+    podman.enabled = false
+    shifter.enabled = false
+    charliecloud.enabled = false
+    process.conda = "$projectDir/environment.yml"
+  }
   debug { process.beforeScript = 'echo $HOSTNAME' }
   docker {
     docker.enabled = true
+    singularity.enabled = false
+    podman.enabled = false
+    shifter.enabled = false
+    charliecloud.enabled = false
     // Avoid this error:
     //   WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
     // Testing this in nf-core after discussion here https://github.com/nf-core/tools/pull/351
     // once this is established and works well, nextflow might implement this behavior as new default.
     docker.runOptions = '-u \$(id -u):\$(id -g)'
```

Comment:
There was this discussion here https://github.com/nf-core/modules/pull/525 on using docker.userEmulation instead. I don't know if this is common now or not, but just wanted to leave it here for discussion.

Reply:
Ha, never heard about that one  - we should have that in the template I guess 😓

Reply:
Found it: https://github.com/nf-core/tools/pull/1095 I guess it is

Reply:
Thanks, added now!